.github/workflows/fork-ci.yaml

```diff
@@ -102,18 +107,18 @@ jobs:
 
           git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
 
-          # NOTE: ignore Chart.yaml changes because we refer to chart name in values.yaml and have hardcoded version.
-          cp -r ../sriov-network-operator-fork/deployment/sriov-network-operator-chart/crds/ deployment/network-operator/charts/sriov-network-operator/
-          cp -r ../sriov-network-operator-fork/deployment/sriov-network-operator-chart/templates/ deployment/network-operator/charts/sriov-network-operator/
-          cp  ../sriov-network-operator-fork/deployment/sriov-network-operator-chart/README.md deployment/network-operator/charts/sriov-network-operator/
-          cp  ../sriov-network-operator-fork/deployment/sriov-network-operator-chart/values.yaml deployment/network-operator/charts/sriov-network-operator/
+          # we *don't* copy `Chart.yaml` with the files below, because network-operator's `Chart.yaml` refers to the SR-IOV chart name with a hardcoded version.
```

Comment:
Not related to this PR, but we should delete the whole dir and then copy. This is needed in case a file has been deleted from the source.

Reply:
We could also use `rsync --delete` to see all the files that are copied and deleted